site/content/usage/06-vpc-networking.md

```diff
@@ -160,3 +160,40 @@ See the complete example [here](https://github.com/weaveworks/eksctl/blob/master
 
 **Note**: Specifying the NAT Gateway is only supported during cluster creation and it is not touched during a cluster
 upgrade. There are plans to support changing between different modes on cluster update in the future.
+
+### Managing Access to the Kubernetes API Server Endpoints
+
+The default creation of an EKS cluster exposes the Kubernetes API server publicly but not directly from within the
+VPC subnets (Public=true, Private=false). Traffic destined for the API server from within the VPC must first exit the
```

Comment:
there is an inconsistency with capitalized and lowercase public/Public. unless they refer to an actual setting that is capitalized (and then it should probably be in `backticks`) i think lowercase makes more sense